compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/extended/EmptyRangeChecker.kt

```diff
@@ -41,4 +41,28 @@ object EmptyRangeChecker : FirBasicExpresionChecker() {
         }
     }
 
+    private val FirFunctionCall.rangeLeft: Long?
```

Comment:
Add comment for proper integer operator calls checking

Reply:
This comment was added few lines before